### PR TITLE
Add badge for [PackagistStars] (favers)

### LINKED ISSUE #4912: Add badge for Packagist stars (favers)
:clipboard: **Description**

Feature Idea: Add a badge to show stars given on Packagist. Like `packagist stars | 19124` (see Rating section for other examples).

:link: **Data**

Packagist API: Docs https://packagist.org/apidoc - Endpoint `https://packagist.org/packages/<USER>/<PACKAGE>.json`, 

Example page: https://packagist.org/packages/guzzlehttp/guzzle → Stars are shown on the right box

Example API request: https://packagist.org/packages/guzzlehttp/guzzle.json → see `favers`fields

:microphone: **Motivation**

Add a link to package ratings instead of downloads. Since downloads are used to indicate the usefullness of a package… but this is not the case. A package with many stars may be much better than a package with a huge download count.

### PR DESCRIPTION
**Pull request by pixelbrackets**
Add a badge to show stars given to a PHP package on Packagist.

Test:
```
npm run test:services -- --only=packagist
```
New badge UI:
Bildschirmfoto von 2020-05-03 10-57-27

Closes #4912

### CODE REVIEW

File: services/packagist/packagist-stars.service.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+const {
+  keywords,
+  BasePackagistService,
+  customServerDocumentationFragment,
+  cacheDocumentationFragment,
+} = require('./packagist-base')
+
+const schema = Joi.object({
+  package: Joi.object({
+    favers: Joi.number().required(),
```
Comment (PyvesB) [inline services/packagist/packagist-stars.service.js:15]: Can this be replaced by `nonNegativeInteger` in _validators.js_?

File: services/packagist/packagist-stars.tester.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  })
+
+t.create('Stars (valid package, valid custom server)')
+  .get('/guzzlehttp/guzzle.json?server=https%3A%2F%2Fpackagist.org')
+  .expectBadge({
+    label: 'stars',
+    message: isMetric,
+  })
+
+t.create('Stars (invalid package, valid custom server)')
```
Comment (PyvesB) [inline services/packagist/packagist-stars.tester.js:27]: This test seems to be the same as above, the package doesn't seem invalid.

Comment (pixelbrackets) [reply services/packagist/packagist-stars.tester.js:27]: Good catch. Will fix it.

Comment (pixelbrackets) [reply services/packagist/packagist-stars.tester.js:27]: Same in line 42.

File: services/packagist/packagist-stars.service.js (hunk 1 above)
Comment (pixelbrackets) [reply services/packagist/packagist-stars.service.js:15]: Maybe ;) I'll try to find an example usage.

Comment (pixelbrackets) [reply services/packagist/packagist-stars.service.js]: Found some other places where this validator is used, tried it → works.

**Pull request merged**